mozci/push.py

```diff
@@ -1190,34 +1190,28 @@ def classify_regressions(
         else:
             all_groups = push_groups
 
-        groups_relevant_failure = {
+        groups_consistent_failure = {
             g.name
             for g in all_groups
-            if (
-                cross_config_counts is not None
-                and g.is_cross_config_failure(cross_config_counts[1])
-            )
-            or (
-                consistent_failures_counts is not None
-                and g.is_config_consistent_failure(consistent_failures_counts[1])
+            if g.is_consistent_failure(
+                cross_config_counts[1] if cross_config_counts is not None else None,
+                consistent_failures_counts[1]
+                if consistent_failures_counts is not None
+                else None,
             )
         }
 
-        groups_non_relevant_failure = {
+        groups_non_consistent_failure = {
             g.name
             for g in all_groups
             if g.status != Status.PASS
-            and (
-                (
-                    cross_config_counts is None
-                    or g.is_cross_config_failure(cross_config_counts[0]) is False
-                )
-                and (
-                    consistent_failures_counts is None
-                    or g.is_config_consistent_failure(consistent_failures_counts[0])
-                    is False
-                )
```

Comment:
The main logic change is here, where we previously were considering a group as "not relevant failure" / "not consistent failure" only if:
- it was run in multiple configs and it failed on some configs and passed on others;
- it was run multiple times in the same config and it failed on some runs and passed on others.

After this change, we won't need both conditions to be true, only one.